safe-global/safe-wallet-web
# fix: do not show nonce conflict warning for batch transactions.

## What it solves

Resolves #3502 

## How this PR fixes it
Removes the conflicting nonce header for pending transactions.

## How to test it
- Create a batch tx and sign
- Create any other tx and sign
- bulk execute the two Txs
- There should be no warning about conflicting nonces within the bulk tx, as they are all executed as one tx with the same nonce


## Screenshots
<img width="942" alt="image" src="https://github.com/safe-global/safe-wallet-web/assets/17801424/89b285ff-c85d-4f5f-9293-d17c4208ade3">


## Checklist
